diff -up cairo-1.10.0/configure.poppler14 cairo-1.10.0/configure
--- cairo-1.10.0/configure.poppler14	2010-11-11 09:57:19.897533814 -0500
+++ cairo-1.10.0/configure	2010-11-11 09:58:00.193413553 -0500
@@ -30616,7 +30616,7 @@ POPPLER_VERSION_REQUIRED=0.13.3
 test_pdf=no
 any2ppm_pdf=no
 if test "x$use_pdf" = "xyes"; then
-  poppler_DEPENDENCY="poppler-glib >= $POPPLER_VERSION_REQUIRED"
+  poppler_DEPENDENCY="poppler14-glib >= $POPPLER_VERSION_REQUIRED"
 
 pkg_failed=no
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for POPPLER" >&5
